<html>
<head>
<title>TestNG:  API Test</title>
<link href="../testng.css" rel="stylesheet" type="text/css" />
<link href="../my-testng.css" rel="stylesheet" type="text/css" />

<style type="text/css">
.log { display: none;} 
.stack-trace { display: none;} 
</style>
<script type="text/javascript">
<!--
function flip(e) {
  current = e.style.display;
  if (current == 'block') {
    e.style.display = 'none';
    return 0;
  }
  else {
    e.style.display = 'block';
    return 1;
  }
}

function toggleBox(szDivId, elem, msg1, msg2)
{
  var res = -1;  if (document.getElementById) {
    res = flip(document.getElementById(szDivId));
  }
  else if (document.all) {
    // this is the way old msie versions work
    res = flip(document.all[szDivId]);
  }
  if(elem) {
    if(res == 0) elem.innerHTML = msg1; else elem.innerHTML = msg2;
  }

}

function toggleAllBoxes() {
  if (document.getElementsByTagName) {
    d = document.getElementsByTagName('div');
    for (i = 0; i < d.length; i++) {
      if (d[i].className == 'log') {
        flip(d[i]);
      }
    }
  }
}

// -->
</script>

</head>
<body>
<h2 align='center'>API Test</h2><table border='1' align="center">
<tr>
<td>Tests passed/Failed/Skipped:</td><td>36/19/18</td>
</tr><tr>
<td>Started on:</td><td>Tue Apr 06 15:12:10 IST 2021</td>
</tr>
<tr><td>Total time:</td><td>199 seconds (199331 ms)</td>
</tr><tr>
<td>Included groups:</td><td></td>
</tr><tr>
<td>Excluded groups:</td><td></td>
</tr>
</table><p/>
<small><i>(Hover the method name to see the test class name)</i></small><p/>
<table width='100%' border='1' class='invocation-failed'>
<tr><td colspan='4' align='center'><b>FAILED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='TestCases.Car_Model_Ranking.car_basison_ranking()'><b>car_basison_ranking</b><br>Test class: TestCases.Car_Model_Ranking<br>Parameters: {expectedMessage=, expectedErrorCode=200.0, endpoint=qa/api/subscribe/vehicle/model/1?resolution=hdpi}</td>
<td><div><pre>java.lang.IllegalArgumentException: wrong number of arguments
... Removed 25 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1888750776", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1888750776'><pre>java.lang.IllegalArgumentException: wrong number of arguments
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>0</td>
<td>TestCases.Car_Model_Ranking@62e20a76</td></tr>
<tr>
<td title='TestCases.Model_Hero_Image_API_TC.car_basison_ranking()'><b>car_basison_ranking</b><br>Test class: TestCases.Model_Hero_Image_API_TC<br>Parameters: {expectedMessage=, expectedErrorCode=200.0, endpoint=qa/api/subscribe/vehicle/model/1?resolution=hdpi}</td>
<td><div><pre>java.lang.IllegalArgumentException: wrong number of arguments
... Removed 25 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace51432525", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace51432525'><pre>java.lang.IllegalArgumentException: wrong number of arguments
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>0</td>
<td>TestCases.Model_Hero_Image_API_TC@1477089c</td></tr>
<tr>
<td title='TestCases.Customer_Update.customer_Update()'><b>customer_Update</b><br>Test class: TestCases.Customer_Update<br>Parameters: {referralCode=, expectedErrorCode=201, referredBy=, occupationTypeCode=SELF, middleName=Bhardwaj, email=, expectedMessage=null, gender=F, dob=1983-10-26, firstName=Kamini, lastName=, endpoint=qa/customer/api/v1/user/update}</td>
<td><div><pre>java.lang.IllegalArgumentException: wrong number of arguments
... Removed 25 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1268473253", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1268473253'><pre>java.lang.IllegalArgumentException: wrong number of arguments
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>0</td>
<td>TestCases.Customer_Update@7b84fcf8</td></tr>
<tr>
<td title='TestCases.Customer_Update.customer_Update()'><b>customer_Update</b><br>Test class: TestCases.Customer_Update<br>Parameters: {referralCode=, expectedErrorCode=201, referredBy=, occupationTypeCode=SELF, middleName=Singh, email=, expectedMessage=null, gender=F, dob=1983-10-26, firstName=Kamini, lastName=Bhardwaj, endpoint=qa/customer/api/v1/user/update}, org.testng.TestRunner@7dee835</td>
<td><div><pre>java.lang.AssertionError: expected [Kamini] but found [null]
	at com.MSIL.TestUtils.Verify_Response.verify_Response_data(Verify_Response.java:52)
	at com.MSIL.API_LISTS.Cust_Regsiration_API.customer_update_API(Cust_Regsiration_API.java:59)
	at TestCases.Customer_Update.customer_Update(Customer_Update.java:21)
... Removed 31 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1661745907", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1661745907'><pre>java.lang.AssertionError: expected [Kamini] but found [null]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:129)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:189)
	at org.testng.Assert.assertEquals(Assert.java:199)
	at com.MSIL.TestUtils.Verify_Response.verify_Response_data(Verify_Response.java:52)
	at com.MSIL.API_LISTS.Cust_Regsiration_API.customer_update_API(Cust_Regsiration_API.java:59)
	at TestCases.Customer_Update.customer_Update(Customer_Update.java:21)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>0</td>
<td>TestCases.Customer_Update@7b84fcf8</td></tr>
<tr>
<td title='TestCases.Customer_Update.customer_Update()'><b>customer_Update</b><br>Test class: TestCases.Customer_Update<br>Parameters: {referralCode=, expectedErrorCode=201, referredBy=, occupationTypeCode=SELF, middleName=Singh, email=, expectedMessage=null, gender=F, dob=1983-10-26, firstName=Kamini, lastName=Bhardwaj, endpoint=qa/customer/api/v1/user/update}</td>
<td><div><pre>java.lang.IllegalArgumentException: wrong number of arguments
... Removed 25 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace520171591", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace520171591'><pre>java.lang.IllegalArgumentException: wrong number of arguments
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>0</td>
<td>TestCases.Customer_Update@7b84fcf8</td></tr>
<tr>
<td title='TestCases.Fectch_Nearest_Dealer.fetch_nearest_dealer()'><b>fetch_nearest_dealer</b><br>Test class: TestCases.Fectch_Nearest_Dealer<br>Parameters: {expectedMessage=, expectedErrorCode=200.0, pinCode=324324, longitude=72.454545, citycode=1, endpoint=qa/api/partner/nearest-dealers, distance=50000, latitude=19.11454}</td>
<td><div><pre>java.lang.IllegalArgumentException: wrong number of arguments
... Removed 25 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace951517261", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace951517261'><pre>java.lang.IllegalArgumentException: wrong number of arguments
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>0</td>
<td>TestCases.Fectch_Nearest_Dealer@44b3606b</td></tr>
<tr>
<td title='TestCases.Get_Customer_Detail_BY_UUID.get_customer_detail_uuid()'><b>get_customer_detail_uuid</b><br>Test class: TestCases.Get_Customer_Detail_BY_UUID<br>Parameters: {expectedMessage=, expectedErrorCode=200, endpoint=qa/customer/api/v1/user/details/:uuid=d9b97efd-69b7-4e6b-8d10-f4b96a81d9ed}</td>
<td><div><pre>java.lang.IllegalArgumentException: wrong number of arguments
... Removed 25 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace953354471", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace953354471'><pre>java.lang.IllegalArgumentException: wrong number of arguments
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>0</td>
<td>TestCases.Get_Customer_Detail_BY_UUID@56f0cc85</td></tr>
<tr>
<td title='TestCases.Customer_Registartion.invalid_customer_registration()'><b>invalid_customer_registration</b><br>Test class: TestCases.Customer_Registartion<br>Parameters: {mobile=1, businessType=SUBSCRIBE, referralCode=, expectedErrorCode=400, referredBy=, occupationTypeCode=SELF, middleName=Kumar, email=gurender.kush@nagarro.com, expectedMessage=JSON parse error: Cannot deserialize value of type `java.time.LocalDate` from String \"26-10-1983\": Failed to deserialize java.time.LocalDate: (java.time.format.DateTimeParseException) Text '26-10-1983' could not be parsed at index 5; nested exception is com.fasterxml.jackson.databind.exc.InvalidFormatException: Cannot deserialize value of type `java.time.LocalDate` from String \"26-10-1983\": Failed to deserialize java.time.LocalDate: (java.time.format.DateTimeParseException) Text '26-10-1983' could not be parsed at index 5\n at [Source: (PushbackInputStream); line: 8, column: 12] (through reference chain: com.marutisuzuki.customer.dto.request.CustomerRegisterReqDto[\"dob\"]), gender=, dob=26-10-1983, firstName=Gurender, lastName=Kush, endpoint=qa/customer/api/v1/register}</td>
<td><div><pre>java.lang.IllegalArgumentException: wrong number of arguments
... Removed 25 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1209290522", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1209290522'><pre>java.lang.IllegalArgumentException: wrong number of arguments
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>0</td>
<td>TestCases.Customer_Registartion@628c4ac0</td></tr>
<tr>
<td title='TestCases.Customer_Registartion.invalid_customer_registration()'><b>invalid_customer_registration</b><br>Test class: TestCases.Customer_Registartion<br>Parameters: {mobile=1, businessType=SUBSCRIBE, referralCode=, expectedErrorCode=400, referredBy=, occupationTypeCode=SELF, middleName=Kumar, email=gurender.kush@nagarro.com, expectedMessage=JSON parse error: Cannot deserialize value of type `java.time.LocalDate` from String \"1983/01/01'\": Failed to deserialize java.time.LocalDate: (java.time.format.DateTimeParseException) Text '1983/01/01' could not be parsed at index 5; nested exception is com.fasterxml.jackson.databind.exc.InvalidFormatException: Cannot deserialize value of type `java.time.LocalDate` from String \"1983/01/01\": Failed to deserialize java.time.LocalDate: (java.time.format.DateTimeParseException) Text '1983/01/01' could not be parsed at index 5\n at [Source: (PushbackInputStream); line: 8, column: 12] (through reference chain: com.marutisuzuki.customer.dto.request.CustomerRegisterReqDto[\"dob\"]), gender=, dob=1983/01/01, firstName=Gurender, lastName=Kush, endpoint=qa/customer/api/v1/register}, org.testng.TestRunner@7dee835</td>
<td><div><pre>java.lang.AssertionError: expected [JSON parse error: Cannot deserialize value of type `java.time.LocalDate` from String &quot;1983/01/01&apos;&quot;: Failed to deserialize java.time.LocalDate: (java.time.format.DateTimeParseException) Text &apos;1983/01/01&apos; could not be parsed at index 5; nested exception is com.fasterxml.jackson.databind.exc.InvalidFormatException: Cannot deserialize value of type `java.time.LocalDate` from String &quot;1983/01/01&quot;: Failed to deserialize java.time.LocalDate: (java.time.format.DateTimeParseException) Text &apos;1983/01/01&apos; could not be parsed at index 5n at [Source: (PushbackInputStream); line: 8, column: 12] (through reference chain: com.marutisuzuki.customer.dto.request.CustomerRegisterReqDto[&quot;dob&quot;])] but found [JSON parse error: Cannot deserialize value of type `java.time.LocalDate` from String &quot;1983/01/01&quot;: Failed to deserialize java.time.LocalDate: (java.time.format.DateTimeParseException) Text &apos;1983/01/01&apos; could not be parsed at index 4; nested exception is com.fasterxml.jackson.databind.exc.InvalidFormatException: Cannot deserialize value of type `java.time.LocalDate` from String &quot;1983/01/01&quot;: Failed to deserialize java.time.LocalDate: (java.time.format.DateTimeParseException) Text &apos;1983/01/01&apos; could not be parsed at index 4
 at [Source: (PushbackInputStream); line: 1, column: 128] (through reference chain: com.marutisuzuki.customer.dto.request.CustomerRegisterReqDto[&quot;dob&quot;])]
	at com.MSIL.TestUtils.Verify_Response.verify_Response_1(Verify_Response.java:38)
	at com.MSIL.API_LISTS.Cust_Regsiration_API.customer_registration_API(Cust_Regsiration_API.java:40)
	at TestCases.Customer_Registartion.invalid_customer_registration(Customer_Registartion.java:33)
... Removed 31 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1468111065", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1468111065'><pre>java.lang.AssertionError: expected [JSON parse error: Cannot deserialize value of type `java.time.LocalDate` from String &quot;1983/01/01&apos;&quot;: Failed to deserialize java.time.LocalDate: (java.time.format.DateTimeParseException) Text &apos;1983/01/01&apos; could not be parsed at index 5; nested exception is com.fasterxml.jackson.databind.exc.InvalidFormatException: Cannot deserialize value of type `java.time.LocalDate` from String &quot;1983/01/01&quot;: Failed to deserialize java.time.LocalDate: (java.time.format.DateTimeParseException) Text &apos;1983/01/01&apos; could not be parsed at index 5n at [Source: (PushbackInputStream); line: 8, column: 12] (through reference chain: com.marutisuzuki.customer.dto.request.CustomerRegisterReqDto[&quot;dob&quot;])] but found [JSON parse error: Cannot deserialize value of type `java.time.LocalDate` from String &quot;1983/01/01&quot;: Failed to deserialize java.time.LocalDate: (java.time.format.DateTimeParseException) Text &apos;1983/01/01&apos; could not be parsed at index 4; nested exception is com.fasterxml.jackson.databind.exc.InvalidFormatException: Cannot deserialize value of type `java.time.LocalDate` from String &quot;1983/01/01&quot;: Failed to deserialize java.time.LocalDate: (java.time.format.DateTimeParseException) Text &apos;1983/01/01&apos; could not be parsed at index 4
 at [Source: (PushbackInputStream); line: 1, column: 128] (through reference chain: com.marutisuzuki.customer.dto.request.CustomerRegisterReqDto[&quot;dob&quot;])]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:189)
	at org.testng.Assert.assertEquals(Assert.java:199)
	at com.MSIL.TestUtils.Verify_Response.verify_Response_1(Verify_Response.java:38)
	at com.MSIL.API_LISTS.Cust_Regsiration_API.customer_registration_API(Cust_Regsiration_API.java:40)
	at TestCases.Customer_Registartion.invalid_customer_registration(Customer_Registartion.java:33)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>0</td>
<td>TestCases.Customer_Registartion@628c4ac0</td></tr>
<tr>
<td title='TestCases.Customer_Registartion.invalid_customer_registration()'><b>invalid_customer_registration</b><br>Test class: TestCases.Customer_Registartion<br>Parameters: {mobile=1, businessType=SUBSCRIBE, referralCode=, expectedErrorCode=400, referredBy=, occupationTypeCode=SELF, middleName=Kumar, email=gurender.kush@nagarro.com, expectedMessage=JSON parse error: Cannot deserialize value of type `java.time.LocalDate` from String \"1983/01/01'\": Failed to deserialize java.time.LocalDate: (java.time.format.DateTimeParseException) Text '1983/01/01' could not be parsed at index 5; nested exception is com.fasterxml.jackson.databind.exc.InvalidFormatException: Cannot deserialize value of type `java.time.LocalDate` from String \"1983/01/01\": Failed to deserialize java.time.LocalDate: (java.time.format.DateTimeParseException) Text '1983/01/01' could not be parsed at index 5\n at [Source: (PushbackInputStream); line: 8, column: 12] (through reference chain: com.marutisuzuki.customer.dto.request.CustomerRegisterReqDto[\"dob\"]), gender=, dob=1983/01/01, firstName=Gurender, lastName=Kush, endpoint=qa/customer/api/v1/register}</td>
<td><div><pre>java.lang.IllegalArgumentException: wrong number of arguments
... Removed 25 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace523938160", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace523938160'><pre>java.lang.IllegalArgumentException: wrong number of arguments
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>0</td>
<td>TestCases.Customer_Registartion@628c4ac0</td></tr>
<tr>
<td title='TestCases.Customer_Registartion.invalid_customer_registration()'><b>invalid_customer_registration</b><br>Test class: TestCases.Customer_Registartion<br>Parameters: {mobile=1, businessType=SUBSCRIBE, referralCode=, expectedErrorCode=400, referredBy=, occupationTypeCode=SELF, middleName=Kumar, email=gurender.kush@nagarro.com, expectedMessage=JSON parse error: Cannot deserialize value of type `java.time.LocalDate` from String \"1983-1-1\": Failed to deserialize java.time.LocalDate: (java.time.format.DateTimeParseException) Text '1983-1-1' could not be parsed at index 5; nested exception is com.fasterxml.jackson.databind.exc.InvalidFormatException: Cannot deserialize value of type `java.time.LocalDate` from String \"1983-1-1\": Failed to deserialize java.time.LocalDate: (java.time.format.DateTimeParseException) Text '1983-1-1' could not be parsed at index 5\n at [Source: (PushbackInputStream); line: 8, column: 12] (through reference chain: com.marutisuzuki.customer.dto.request.CustomerRegisterReqDto[\"dob\"]), gender=, dob=1983-01-01, firstName=Gurender, lastName=Kush, endpoint=qa/customer/api/v1/register}, org.testng.TestRunner@7dee835</td>
<td><div><pre>java.lang.NullPointerException
	at com.MSIL.TestUtils.Verify_Response.verify_Response_1(Verify_Response.java:35)
	at com.MSIL.API_LISTS.Cust_Regsiration_API.customer_registration_API(Cust_Regsiration_API.java:40)
	at TestCases.Customer_Registartion.invalid_customer_registration(Customer_Registartion.java:33)
... Removed 25 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace518137704", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace518137704'><pre>java.lang.NullPointerException
	at com.MSIL.TestUtils.Verify_Response.verify_Response_1(Verify_Response.java:35)
	at com.MSIL.API_LISTS.Cust_Regsiration_API.customer_registration_API(Cust_Regsiration_API.java:40)
	at TestCases.Customer_Registartion.invalid_customer_registration(Customer_Registartion.java:33)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>0</td>
<td>TestCases.Customer_Registartion@628c4ac0</td></tr>
<tr>
<td title='TestCases.Customer_Registartion.invalid_customer_registration()'><b>invalid_customer_registration</b><br>Test class: TestCases.Customer_Registartion<br>Parameters: {mobile=1, businessType=SUBSCRIBE, referralCode=, expectedErrorCode=400, referredBy=, occupationTypeCode=SELF, middleName=Kumar, email=gurender.kush@nagarro.com, expectedMessage=JSON parse error: Cannot deserialize value of type `java.time.LocalDate` from String \"1983-1-1\": Failed to deserialize java.time.LocalDate: (java.time.format.DateTimeParseException) Text '1983-1-1' could not be parsed at index 5; nested exception is com.fasterxml.jackson.databind.exc.InvalidFormatException: Cannot deserialize value of type `java.time.LocalDate` from String \"1983-1-1\": Failed to deserialize java.time.LocalDate: (java.time.format.DateTimeParseException) Text '1983-1-1' could not be parsed at index 5\n at [Source: (PushbackInputStream); line: 8, column: 12] (through reference chain: com.marutisuzuki.customer.dto.request.CustomerRegisterReqDto[\"dob\"]), gender=, dob=1983-01-01, firstName=Gurender, lastName=Kush, endpoint=qa/customer/api/v1/register}</td>
<td><div><pre>java.lang.IllegalArgumentException: wrong number of arguments
... Removed 25 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace721840156", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace721840156'><pre>java.lang.IllegalArgumentException: wrong number of arguments
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>0</td>
<td>TestCases.Customer_Registartion@628c4ac0</td></tr>
<tr>
<td title='TestCases.Customer_Update.invalid_customer_update()'><b>invalid_customer_update</b><br>Test class: TestCases.Customer_Update<br>Parameters: {referralCode=, expectedErrorCode=400, referredBy=, occupationTypeCode=SELF, middleName=Bhardwaj, email=, expectedMessage=First Name is mandatory, gender=F, dob=1983-10-26, firstName=, lastName=, endpoint=qa/customer/api/v1/user/update}</td>
<td><div><pre>java.lang.IllegalArgumentException: wrong number of arguments
... Removed 25 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace832376940", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace832376940'><pre>java.lang.IllegalArgumentException: wrong number of arguments
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>0</td>
<td>TestCases.Customer_Update@7b84fcf8</td></tr>
<tr>
<td title='TestCases.Customer_Update.invalid_customer_update()'><b>invalid_customer_update</b><br>Test class: TestCases.Customer_Update<br>Parameters: {referralCode=, expectedErrorCode=400, referredBy=, occupationTypeCode=SELF, middleName=Bhardwaj, email=, expectedMessage=JSON parse error: Cannot deserialize value of type `java.time.LocalDate` from String "26-10-1983": Failed to deserialize java.time.LocalDate: (java.time.format.DateTimeParseException) Text '26-10-1983' could not be parsed at index 0; nested exception is com.fasterxml.jackson.databind.exc.InvalidFormatException: Cannot deserialize value of type `java.time.LocalDate` from String "26-10-1983": Failed to deserialize java.time.LocalDate: (java.time.format.DateTimeParseException) Text '26-10-1983' could not be parsed at index 0, gender=F, dob=26-10-1983, firstName=Kamini, lastName=, endpoint=qa/customer/api/v1/user/update}, org.testng.TestRunner@7dee835</td>
<td><div><pre>java.lang.NullPointerException
	at com.MSIL.TestUtils.Verify_Response.verify_Response_data_1(Verify_Response.java:65)
	at com.MSIL.API_LISTS.Cust_Regsiration_API.customer_update_API(Cust_Regsiration_API.java:62)
	at TestCases.Customer_Update.invalid_customer_update(Customer_Update.java:32)
... Removed 25 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace2136187679", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace2136187679'><pre>java.lang.NullPointerException
	at com.MSIL.TestUtils.Verify_Response.verify_Response_data_1(Verify_Response.java:65)
	at com.MSIL.API_LISTS.Cust_Regsiration_API.customer_update_API(Cust_Regsiration_API.java:62)
	at TestCases.Customer_Update.invalid_customer_update(Customer_Update.java:32)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>0</td>
<td>TestCases.Customer_Update@7b84fcf8</td></tr>
<tr>
<td title='TestCases.Customer_Update.invalid_customer_update()'><b>invalid_customer_update</b><br>Test class: TestCases.Customer_Update<br>Parameters: {referralCode=, expectedErrorCode=400, referredBy=, occupationTypeCode=SELF, middleName=Bhardwaj, email=, expectedMessage=JSON parse error: Cannot deserialize value of type `java.time.LocalDate` from String "26-10-1983": Failed to deserialize java.time.LocalDate: (java.time.format.DateTimeParseException) Text '26-10-1983' could not be parsed at index 0; nested exception is com.fasterxml.jackson.databind.exc.InvalidFormatException: Cannot deserialize value of type `java.time.LocalDate` from String "26-10-1983": Failed to deserialize java.time.LocalDate: (java.time.format.DateTimeParseException) Text '26-10-1983' could not be parsed at index 0, gender=F, dob=26-10-1983, firstName=Kamini, lastName=, endpoint=qa/customer/api/v1/user/update}</td>
<td><div><pre>java.lang.IllegalArgumentException: wrong number of arguments
... Removed 25 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace547512166", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace547512166'><pre>java.lang.IllegalArgumentException: wrong number of arguments
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>0</td>
<td>TestCases.Customer_Update@7b84fcf8</td></tr>
<tr>
<td title='TestCases.Customer_Update.invalid_customer_update()'><b>invalid_customer_update</b><br>Test class: TestCases.Customer_Update<br>Parameters: {referralCode=, expectedErrorCode=400, referredBy=, occupationTypeCode=ABC, middleName=Bhardwaj, email=, expectedMessage=Occupation type does not exist for given code, gender=F, dob=1983-10-26, firstName=Kamini, lastName=, endpoint=qa/customer/api/v1/user/update}, org.testng.TestRunner@7dee835</td>
<td><div><pre>java.lang.NullPointerException
	at com.MSIL.TestUtils.Verify_Response.verify_Response_data_1(Verify_Response.java:65)
	at com.MSIL.API_LISTS.Cust_Regsiration_API.customer_update_API(Cust_Regsiration_API.java:62)
	at TestCases.Customer_Update.invalid_customer_update(Customer_Update.java:32)
... Removed 25 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1187125406", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1187125406'><pre>java.lang.NullPointerException
	at com.MSIL.TestUtils.Verify_Response.verify_Response_data_1(Verify_Response.java:65)
	at com.MSIL.API_LISTS.Cust_Regsiration_API.customer_update_API(Cust_Regsiration_API.java:62)
	at TestCases.Customer_Update.invalid_customer_update(Customer_Update.java:32)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>0</td>
<td>TestCases.Customer_Update@7b84fcf8</td></tr>
<tr>
<td title='TestCases.Customer_Update.invalid_customer_update()'><b>invalid_customer_update</b><br>Test class: TestCases.Customer_Update<br>Parameters: {referralCode=, expectedErrorCode=400, referredBy=, occupationTypeCode=ABC, middleName=Bhardwaj, email=, expectedMessage=Occupation type does not exist for given code, gender=F, dob=1983-10-26, firstName=Kamini, lastName=, endpoint=qa/customer/api/v1/user/update}</td>
<td><div><pre>java.lang.IllegalArgumentException: wrong number of arguments
... Removed 25 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace105790402", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace105790402'><pre>java.lang.IllegalArgumentException: wrong number of arguments
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>0</td>
<td>TestCases.Customer_Update@7b84fcf8</td></tr>
<tr>
<td title='TestCases.Leasing_Partner_Ranking.leasing_ranking()'><b>leasing_ranking</b><br>Test class: TestCases.Leasing_Partner_Ranking<br>Parameters: {expectedMessage=, expectedErrorCode=200.0, endpoint=qa/api/partner/leasing-company-rating}</td>
<td><div><pre>java.lang.IllegalArgumentException: wrong number of arguments
... Removed 25 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace511033073", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace511033073'><pre>java.lang.IllegalArgumentException: wrong number of arguments
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>0</td>
<td>TestCases.Leasing_Partner_Ranking@2cc44ad</td></tr>
<tr>
<td title='TestCases.Fectch_Nearest_Dealer.nearest_dealer_not_found()'><b>nearest_dealer_not_found</b><br>Test class: TestCases.Fectch_Nearest_Dealer<br>Parameters: {expectedMessage=Dealer not found., expectedErrorCode=400, pinCode=324324, longitude=72.454545, citycode=1, endpoint=qa/api/partner/nearest-dealers, distance=50, latitude=19.11454}</td>
<td><div><pre>java.lang.IllegalArgumentException: wrong number of arguments
... Removed 25 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1893562871", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1893562871'><pre>java.lang.IllegalArgumentException: wrong number of arguments
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>0</td>
<td>TestCases.Fectch_Nearest_Dealer@44b3606b</td></tr>
</table><p>
<table width='100%' border='1' class='invocation-passed'>
<tr><td colspan='4' align='center'><b>PASSED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='TestCases.WhiteList_Ref_isHybrid_TC.Whitelist_Ref_ishybrid()'><b>Whitelist_Ref_ishybrid</b><br>Test class: TestCases.WhiteList_Ref_isHybrid_TC<br>Parameters: {colorsRequired=true, bodyTypes=, expectedErrorCode=200, channelTypesRequired=false, channelTypes=, resolution=xxhdpi, colorCodes=, enginTypesRequired=false, transmissionTypesRequired=false, modelCodes=BA, enginTypes=, expectedMessage=, fuelTypesRequired=false, variantCodes=BAA4CZ1, endpoint=qa/api/subscribe/vehicle/whitelists/model-variant, bodyTypesRequired=false}, org.testng.TestRunner@7dee835</td>
<td></td>
<td>1</td>
<td>TestCases.WhiteList_Ref_isHybrid_TC@34cdeda2</td></tr>
<tr>
<td title='TestCases.Add_Customer_Address.add_customer_address()'><b>add_customer_address</b><br>Test class: TestCases.Add_Customer_Address<br>Parameters: {addressLine3=behind batta, addressLine2=vikranth shastri marg1, addressLine1=H.no-1, addressType=WORK, addressCategory=BILLING, state=UP, endpoint=/qa/customer/api/v1/customer-address, expectedMessage=Addresses added successfully, gstNumber=gst09875456789, longitude=76.3, pincode=767, expectedErrorCode=201, dmsCityCode=ND, latitude=23.90, city=Noida, area=gandhi road, standard=true, cibilStateCode=CUP}, org.testng.TestRunner@7dee835</td>
<td></td>
<td>1</td>
<td>TestCases.Add_Customer_Address@663411de</td></tr>
<tr>
<td title='TestCases.All_Master_Generic_API.all_master_generic_api()'><b>all_master_generic_api</b><br>Test class: TestCases.All_Master_Generic_API<br>Parameters: {expectedErrorCode=200, variant=true, color=true, transmissionType=true, channelType=true, model=true, registrationType=true, expectedMessage=, bodyType=true, fuelType=true, endpoint=qa/api/subscribe/vehicle/vehicle-attributes-master}, org.testng.TestRunner@7dee835</td>
<td></td>
<td>0</td>
<td>TestCases.All_Master_Generic_API@366ac49b</td></tr>
<tr>
<td title='TestCases.City_Master_API_TC.city_master()'><b>city_master</b><br>Test class: TestCases.City_Master_API_TC<br>Parameters: {isNew=true, currentDate=null, expectedMessage=, expectedErrorCode=200, regType=WHITE_PLATE, isFuture=false, endpoint=qa/api/common/city-list, cityId=1}, org.testng.TestRunner@7dee835</td>
<td></td>
<td>1</td>
<td>TestCases.City_Master_API_TC@6ad59d92</td></tr>
<tr>
<td title='TestCases.City_Master_Number_Plate_TC.city_master()'><b>city_master</b><br>Test class: TestCases.City_Master_Number_Plate_TC<br>Parameters: {isNew=true, currentDate=null, expectedMessage=, expectedErrorCode=200, regType=WHITE_PLATE, isFuture=false, endpoint=qa/api/common/city-list, cityId=1}, org.testng.TestRunner@7dee835</td>
<td></td>
<td>0</td>
<td>TestCases.City_Master_Number_Plate_TC@9ef8eb7</td></tr>
<tr>
<td title='TestCases.Customer_Exists.customer_exists()'><b>customer_exists</b><br>Test class: TestCases.Customer_Exists<br>Parameters: {mobile=9669483959, expectedMessage=, expectedErrorCode=200, endpoint=qa/customer/api/v1/exists}, org.testng.TestRunner@7dee835</td>
<td></td>
<td>0</td>
<td>TestCases.Customer_Exists@30b19518</td></tr>
<tr>
<td title='TestCases.Customer_Register_Cognito.customer_register_cognito()'><b>customer_register_cognito</b><br>Test class: TestCases.Customer_Register_Cognito<br>Parameters: {uuid=, mobile=1, businessType=SUBSCRIBE, referralCode=, expectedErrorCode=200, referredBy=, occupationTypeCode=, middleName=Kumar, email=gurender.kush@nagarro.com, expectedMessage=Customer successfully registered, gender=M, dob=1983-10-26, firstName=Gurender, lastName=Kush, endpoint=qa/customer/api/v1/user/register}, org.testng.TestRunner@7dee835</td>
<td></td>
<td>19</td>
<td>TestCases.Customer_Register_Cognito@1b73be9f</td></tr>
<tr>
<td title='TestCases.Customer_Registartion.customer_registration()'><b>customer_registration</b><br>Test class: TestCases.Customer_Registartion<br>Parameters: {mobile=1, businessType=SUBSCRIBE, referralCode=, expectedErrorCode=201.0, referredBy=, occupationTypeCode=SELF, middleName=Kumar, email=gurender.kush@nagarro.com, expectedMessage=Customer successfully registered, gender=, dob=, firstName=Gurender, lastName=Kush, endpoint=qa/customer/api/v1/register}, org.testng.TestRunner@7dee835</td>
<td></td>
<td>0</td>
<td>TestCases.Customer_Registartion@628c4ac0</td></tr>
<tr>
<td title='TestCases.Customer_Registartion.customer_registration()'><b>customer_registration</b><br>Test class: TestCases.Customer_Registartion<br>Parameters: {mobile=1, businessType=SUBSCRIBE, referralCode=, expectedErrorCode=201.0, referredBy=, occupationTypeCode=SELF, middleName=Kumar, email=, expectedMessage=Customer successfully registered, gender=, dob=, firstName=Gurender, lastName=Kush, endpoint=qa/customer/api/v1/register}, org.testng.TestRunner@7dee835</td>
<td></td>
<td>1</td>
<td>TestCases.Customer_Registartion@628c4ac0</td></tr>
<tr>
<td title='TestCases.Customer_Registartion.customer_registration()'><b>customer_registration</b><br>Test class: TestCases.Customer_Registartion<br>Parameters: {mobile=1, businessType=SUBSCRIBE, referralCode=, expectedErrorCode=201.0, referredBy=, occupationTypeCode=, middleName=Kumar, email=gurender.kush@nagarro.com, expectedMessage=Customer successfully registered, gender=, dob=, firstName=Gurender, lastName=Kush, endpoint=qa/customer/api/v1/register}, org.testng.TestRunner@7dee835</td>
<td></td>
<td>0</td>
<td>TestCases.Customer_Registartion@628c4ac0</td></tr>
<tr>
<td title='TestCases.Customer_Registartion.customer_registration()'><b>customer_registration</b><br>Test class: TestCases.Customer_Registartion<br>Parameters: {mobile=1, businessType=SUBSCRIBE, referralCode=, expectedErrorCode=201.0, referredBy=, occupationTypeCode=SELF, middleName=Kumar, email=gurender.kush@nagarro.com, expectedMessage=Customer successfully registered, gender=M, dob=1983-10-26, firstName=Gurender, lastName=Kush, endpoint=qa/customer/api/v1/register}, org.testng.TestRunner@7dee835</td>
<td></td>
<td>0</td>
<td>TestCases.Customer_Registartion@628c4ac0</td></tr>
<tr>
<td title='TestCases.Customer_Registartion.customer_registration()'><b>customer_registration</b><br>Test class: TestCases.Customer_Registartion<br>Parameters: {mobile=1, businessType=SUBSCRIBE, referralCode=, expectedErrorCode=201.0, referredBy=, occupationTypeCode=SELF, middleName=Kumar, email=   gurender.kush@nagarro.com, expectedMessage=Customer successfully registered, gender=, dob=1983-10-26, firstName=Gurender, lastName=Kush, endpoint=qa/customer/api/v1/register}, org.testng.TestRunner@7dee835</td>
<td></td>
<td>1</td>
<td>TestCases.Customer_Registartion@628c4ac0</td></tr>
<tr>
<td title='TestCases.Customer_Registartion.customer_registration()'><b>customer_registration</b><br>Test class: TestCases.Customer_Registartion<br>Parameters: {mobile=1, businessType=SUBSCRIBE, referralCode=, expectedErrorCode=201.0, referredBy=, occupationTypeCode=SELF, middleName=Kumar, email=   gurender.kush@nagarro.com   , expectedMessage=Customer successfully registered, gender=, dob=1983-10-26, firstName=Gurender, lastName=Kush, endpoint=qa/customer/api/v1/register}, org.testng.TestRunner@7dee835</td>
<td></td>
<td>1</td>
<td>TestCases.Customer_Registartion@628c4ac0</td></tr>
<tr>
<td title='TestCases.Customer_Registartion.customer_registration()'><b>customer_registration</b><br>Test class: TestCases.Customer_Registartion<br>Parameters: {mobile=1, businessType=SUBSCRIBE, referralCode=1.0, expectedErrorCode=201.0, referredBy=Gurender, occupationTypeCode=SELF, middleName=Kumar, email=   gurender.kush@nagarro.com   , expectedMessage=Customer successfully registered, gender=M, dob=1983-10-26, firstName=Gurender, lastName=Kush, endpoint=qa/customer/api/v1/register}, org.testng.TestRunner@7dee835</td>
<td></td>
<td>5</td>
<td>TestCases.Customer_Registartion@628c4ac0</td></tr>
<tr>
<td title='TestCases.Customer_Register_Send_Otp.customer_regsiter_send_otp()'><b>customer_regsiter_send_otp</b><br>Test class: TestCases.Customer_Register_Send_Otp<br>Parameters: {mobile=1, expectedMessage=Otp sent successfully, expectedErrorCode=200, endpoint=qa/customer/api/v1/otp/register/send}, org.testng.TestRunner@7dee835</td>
<td></td>
<td>52</td>
<td>TestCases.Customer_Register_Send_Otp@604f2bd2</td></tr>
<tr>
<td title='TestCases.Common_Cust_Verify_Otp.customer_verify_otp()'><b>customer_verify_otp</b><br>Test class: TestCases.Common_Cust_Verify_Otp<br>Parameters: {contactInfo=1, expectedMessage=null, otp=1, expectedErrorCode=200, endpoint=qa/customer/api/v1/otp/verify}, org.testng.TestRunner@7dee835</td>
<td></td>
<td>0</td>
<td>TestCases.Common_Cust_Verify_Otp@1d3ac898</td></tr>
<tr>
<td title='TestCases.generateToken.gen_auth_token()'><b>gen_auth_token</b><br>Test class: TestCases.generateToken<br>Parameters: {mobileNo=9686316676, otp=, appId=4, endpoint=qa/api/account/otp/authtoken}, org.testng.TestRunner@7dee835</td>
<td></td>
<td>3</td>
<td>TestCases.generateToken@1e8ce150</td></tr>
<tr>
<td title='TestCases.CustomerAddressGet.get_customer_address()'><b>get_customer_address</b><br>Test class: TestCases.CustomerAddressGet<br>Parameters: {expectedMessage=, expectedErrorCode=200, endpoint=/qa/customer/api/v1/customer-address/}, org.testng.TestRunner@7dee835</td>
<td></td>
<td>3</td>
<td>TestCases.CustomerAddressGet@63dd899</td></tr>
<tr>
<td title='TestCases.Get_Customer_Details.get_customer_details()'><b>get_customer_details</b><br>Test class: TestCases.Get_Customer_Details<br>Parameters: {expectedMessage=, expectedErrorCode=200, endpoint=qa/customer/api/v1/user/details}, org.testng.TestRunner@7dee835</td>
<td></td>
<td>68</td>
<td>TestCases.Get_Customer_Details@363042d7</td></tr>
<tr>
<td title='TestCases.GetCustomerOccupationById.get_customer_occupation()'><b>get_customer_occupation</b><br>Test class: TestCases.GetCustomerOccupationById<br>Parameters: {expectedMessage=, expectedErrorCode=201, endpoint=qa/customer/api/v1/employment}, org.testng.TestRunner@7dee835</td>
<td></td>
<td>1</td>
<td>TestCases.GetCustomerOccupationById@5b202a3a</td></tr>
<tr>
<td title='TestCases.GetCustomerByUUIDAndOccupationId.get_customer_occupation_UUID_ID()'><b>get_customer_occupation_UUID_ID</b><br>Test class: TestCases.GetCustomerByUUIDAndOccupationId<br>Parameters: {expectedMessage=, expectedErrorCode=201, endpoint=qa/customer/api/v1}, org.testng.TestRunner@7dee835</td>
<td></td>
<td>0</td>
<td>TestCases.GetCustomerByUUIDAndOccupationId@10b9db7b</td></tr>
<tr>
<td title='TestCases.Customer_Exists.invalid_customer_exists()'><b>invalid_customer_exists</b><br>Test class: TestCases.Customer_Exists<br>Parameters: {mobile=, expectedMessage=Please provide valid mobile, expectedErrorCode=400, endpoint=qa/customer/api/v1/exists}, org.testng.TestRunner@7dee835</td>
<td></td>
<td>0</td>
<td>TestCases.Customer_Exists@30b19518</td></tr>
<tr>
<td title='TestCases.Customer_Exists.invalid_customer_exists()'><b>invalid_customer_exists</b><br>Test class: TestCases.Customer_Exists<br>Parameters: {mobile=12345, expectedMessage=Please provide valid mobile, expectedErrorCode=400, endpoint=qa/customer/api/v1/exists}, org.testng.TestRunner@7dee835</td>
<td></td>
<td>0</td>
<td>TestCases.Customer_Exists@30b19518</td></tr>
<tr>
<td title='TestCases.Customer_Exists.invalid_customer_exists()'><b>invalid_customer_exists</b><br>Test class: TestCases.Customer_Exists<br>Parameters: {mobile=9111111111, expectedMessage=Customer not exists., expectedErrorCode=400, endpoint=qa/customer/api/v1/exists}, org.testng.TestRunner@7dee835</td>
<td></td>
<td>0</td>
<td>TestCases.Customer_Exists@30b19518</td></tr>
<tr>
<td title='TestCases.Customer_Registartion.invalid_customer_registration()'><b>invalid_customer_registration</b><br>Test class: TestCases.Customer_Registartion<br>Parameters: {mobile=, businessType=SUBSCRIBE, referralCode=, expectedErrorCode=400, referredBy=, occupationTypeCode=SELF, middleName=Kumar, email=gurender.kush@nagarro.com, expectedMessage=Please provide valid mobile, gender=, dob=, firstName=Gurender, lastName=Kush, endpoint=qa/customer/api/v1/register}, org.testng.TestRunner@7dee835</td>
<td></td>
<td>0</td>
<td>TestCases.Customer_Registartion@628c4ac0</td></tr>
<tr>
<td title='TestCases.Customer_Registartion.invalid_customer_registration()'><b>invalid_customer_registration</b><br>Test class: TestCases.Customer_Registartion<br>Parameters: {mobile=1, businessType=, referralCode=, expectedErrorCode=400, referredBy=, occupationTypeCode=SELF, middleName=Kumar, email=gurender.kush@nagarro.com, expectedMessage=Business type is mandatory, gender=, dob=, firstName=Gurender, lastName=Kush, endpoint=qa/customer/api/v1/register}, org.testng.TestRunner@7dee835</td>
<td></td>
<td>0</td>
<td>TestCases.Customer_Registartion@628c4ac0</td></tr>
<tr>
<td title='TestCases.Customer_Registartion.invalid_customer_registration()'><b>invalid_customer_registration</b><br>Test class: TestCases.Customer_Registartion<br>Parameters: {mobile=1, businessType=SUBSCRIBE, referralCode=, expectedErrorCode=400, referredBy=, occupationTypeCode=ABC, middleName=Kumar, email=gurender.kush@nagarro.com, expectedMessage=Occupation type does not exist for given code, gender=, dob=, firstName=Gurender, lastName=Kush, endpoint=qa/customer/api/v1/register}, org.testng.TestRunner@7dee835</td>
<td></td>
<td>1</td>
<td>TestCases.Customer_Registartion@628c4ac0</td></tr>
<tr>
<td title='TestCases.Customer_Registartion.invalid_customer_registration()'><b>invalid_customer_registration</b><br>Test class: TestCases.Customer_Registartion<br>Parameters: {mobile=1, businessType=ABC, referralCode=, expectedErrorCode=400, referredBy=, occupationTypeCode=, middleName=Kumar, email=gurender.kush@nagarro.com, expectedMessage=Please provide valid business type, gender=, dob=, firstName=Gurender, lastName=Kush, endpoint=qa/customer/api/v1/register}, org.testng.TestRunner@7dee835</td>
<td></td>
<td>0</td>
<td>TestCases.Customer_Registartion@628c4ac0</td></tr>
<tr>
<td title='TestCases.Customer_Registartion.invalid_customer_registration()'><b>invalid_customer_registration</b><br>Test class: TestCases.Customer_Registartion<br>Parameters: {mobile=1, businessType=SUBSCRIBE, referralCode=, expectedErrorCode=400, referredBy=, occupationTypeCode=, middleName=Kumar, email=gurender.kush@nagarro.com, expectedMessage=First Name is mandatory, gender=, dob=, firstName=, lastName=Kush, endpoint=qa/customer/api/v1/register}, org.testng.TestRunner@7dee835</td>
<td></td>
<td>0</td>
<td>TestCases.Customer_Registartion@628c4ac0</td></tr>
<tr>
<td title='TestCases.Customer_Register_Send_Otp.invalid_mobile_send_otp()'><b>invalid_mobile_send_otp</b><br>Test class: TestCases.Customer_Register_Send_Otp<br>Parameters: {mobile=, expectedMessage=Please provide valid mobile, expectedErrorCode=400, endpoint=qa/customer/api/v1/otp/register/send}, org.testng.TestRunner@7dee835</td>
<td></td>
<td>0</td>
<td>TestCases.Customer_Register_Send_Otp@604f2bd2</td></tr>
<tr>
<td title='TestCases.Send_OTP.s_otp()'><b>s_otp</b><br>Test class: TestCases.Send_OTP<br>Parameters: {mobileNo=, appId=4.0, endpoint=qa/sendotp}</td>
<td></td>
<td>2</td>
<td>TestCases.Send_OTP@77b14724</td></tr>
<tr>
<td title='TestCases.SaveCustomerOccupation.save_customer_occupation()'><b>save_customer_occupation</b><br>Test class: TestCases.SaveCustomerOccupation<br>Parameters: {workEx=SALES, expectedErrorCode=201, occupationTypeCode=SELF, netIncome=120000, grossIncome=100000, validTill=2031-02-18, employerType=PUBLIC, expectedMessage=, validFrom=2021-02-12, endpoint=qa/customer/api/v1/employment, companyName=MarutiSuzuki}, org.testng.TestRunner@7dee835</td>
<td></td>
<td>1</td>
<td>TestCases.SaveCustomerOccupation@75cd8043</td></tr>
<tr>
<td title='TestCases.UpdateCustomerAddress.update_customer_address()'><b>update_customer_address</b><br>Test class: TestCases.UpdateCustomerAddress<br>Parameters: {addressLine3=behind batta, addressLine2=vikranth shastri marg1, addressLine1=H.no-2, addressType=WORK, addressCategory=BILLING, state=UP, endpoint=/qa/customer/api/v1/customer-address, expectedMessage=Address updated successfully, gstNumber=gst09875456789, longitude=76.3, pincode=767, expectedErrorCode=200, dmsCityCode=ND, latitude=23.90, id=1, city=Noida, area=gandhi road, standard=true, cibilStateCode=CUP}, org.testng.TestRunner@7dee835</td>
<td></td>
<td>1</td>
<td>TestCases.UpdateCustomerAddress@59d2400d</td></tr>
<tr>
<td title='TestCases.UpdateCustomerOccupation.update_customer_occupation()'><b>update_customer_occupation</b><br>Test class: TestCases.UpdateCustomerOccupation<br>Parameters: {workEx=FINANCE, expectedErrorCode=201, occupationTypeCode=SALARIED, netIncome=120000, grossIncome=100000, validTill=2031-02-18, employerType=PUBLIC, expectedMessage=, validFrom=2021-02-12, endpoint=qa/customer/api/v1/employment, companyName=MarutiSuzuki}, org.testng.TestRunner@7dee835</td>
<td></td>
<td>0</td>
<td>TestCases.UpdateCustomerOccupation@33b1c5c5</td></tr>
<tr>
<td title='TestCases.verify_vehicle_List_TC.verify_list()'><b>verify_list</b><br>Test class: TestCases.verify_vehicle_List_TC<br>Parameters: {channelType=, filterSortOnPostGST=true, variantCode=, toPrice=null, popularity=ASC, startDate=null, mileageCode=null, leasingCompCode=null, endpoint=qa/api/subscribe/vehicle/v2/list, fuelType=, expectedErrorCode=200, bodyType=, fromPrice=null, modelCode=SR, forCode=null, tenureId=, resolution=xxhdpi, price=DESC, cityId=8, categoryIds=, transmissionType=, registrationType=BLACK_PLATE, isNew=true, newAddition=ASC, colorType=}, org.testng.TestRunner@7dee835</td>
<td></td>
<td>1</td>
<td>TestCases.verify_vehicle_List_TC@6ee660fb</td></tr>
<tr>
<td title='TestCases.verify_vehicle_List_TC.verify_list_city_invalid()'><b>verify_list_city_invalid</b><br>Test class: TestCases.verify_vehicle_List_TC<br>Parameters: {channelType=, filterSortOnPostGST=true, variantCode=, toPrice=null, popularity=ASC, startDate=null, endpoint=qa/api/subscribe/vehicle/v2/list, fuelType=, expectedErrorCode=, bodyType=, fromPrice=null, modelCode=SR, tenureId=, resolution=xxhdpi, price=DESC, cityId=, categoryIds=5, transmissionType=, registrationType=BLACK_PLATE, isNew=true, newAddition=ASC, colorType=}, org.testng.TestRunner@7dee835</td>
<td></td>
<td>1</td>
<td>TestCases.verify_vehicle_List_TC@6ee660fb</td></tr>
</table><p>
<table width='100%' border='1' class='invocation-skipped'>
<tr><td colspan='4' align='center'><b>SKIPPED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='TestCases.Car_Model_Ranking.car_basison_ranking()'><b>car_basison_ranking</b><br>Test class: TestCases.Car_Model_Ranking<br>Parameters: {expectedMessage=, expectedErrorCode=200.0, endpoint=qa/api/subscribe/vehicle/model/1?resolution=hdpi}, org.testng.TestRunner@7dee835</td>
<td><div><pre>java.lang.AssertionError: expected [14016.0] but found [15663.0]
	at com.MSIL.Response.Model_Ranking_response.verify_Response(Model_Ranking_response.java:37)
	at com.MSIL.API_LISTS.Car_Price_On_Ranking.car_price_on_ranking_API(Car_Price_On_Ranking.java:39)
	at TestCases.Car_Model_Ranking.car_basison_ranking(Car_Model_Ranking.java:19)
... Removed 30 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1704641164", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1704641164'><pre>java.lang.AssertionError: expected [14016.0] but found [15663.0]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:178)
	at com.MSIL.Response.Model_Ranking_response.verify_Response(Model_Ranking_response.java:37)
	at com.MSIL.API_LISTS.Car_Price_On_Ranking.car_price_on_ranking_API(Car_Price_On_Ranking.java:39)
	at TestCases.Car_Model_Ranking.car_basison_ranking(Car_Model_Ranking.java:19)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>1</td>
<td>TestCases.Car_Model_Ranking@62e20a76</td></tr>
<tr>
<td title='TestCases.Car_Model_Ranking.car_basison_ranking()'><b>car_basison_ranking</b><br>Test class: TestCases.Car_Model_Ranking<br>Parameters: {expectedMessage=, expectedErrorCode=200.0, endpoint=qa/api/subscribe/vehicle/model/1?resolution=hdpi}</td>
<td><div><pre>java.lang.IllegalArgumentException: wrong number of arguments
... Removed 25 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace602828967", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace602828967'><pre>java.lang.IllegalArgumentException: wrong number of arguments
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>0</td>
<td>TestCases.Car_Model_Ranking@62e20a76</td></tr>
<tr>
<td title='TestCases.Model_Hero_Image_API_TC.car_basison_ranking()'><b>car_basison_ranking</b><br>Test class: TestCases.Model_Hero_Image_API_TC<br>Parameters: {expectedMessage=, expectedErrorCode=200.0, endpoint=qa/api/subscribe/vehicle/model/1?resolution=hdpi}, org.testng.TestRunner@7dee835</td>
<td><div><pre>java.lang.AssertionError: expected [baleno_back] but found [Cars]
	at com.MSIL.Response.Verify_Response_Hero_Image.verify_Response(Verify_Response_Hero_Image.java:43)
	at com.MSIL.API_LISTS.Car_Price_On_Ranking.hero_image(Car_Price_On_Ranking.java:69)
	at TestCases.Model_Hero_Image_API_TC.car_basison_ranking(Model_Hero_Image_API_TC.java:22)
... Removed 31 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1096030628", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1096030628'><pre>java.lang.AssertionError: expected [baleno_back] but found [Cars]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:189)
	at org.testng.Assert.assertEquals(Assert.java:199)
	at com.MSIL.Response.Verify_Response_Hero_Image.verify_Response(Verify_Response_Hero_Image.java:43)
	at com.MSIL.API_LISTS.Car_Price_On_Ranking.hero_image(Car_Price_On_Ranking.java:69)
	at TestCases.Model_Hero_Image_API_TC.car_basison_ranking(Model_Hero_Image_API_TC.java:22)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>0</td>
<td>TestCases.Model_Hero_Image_API_TC@1477089c</td></tr>
<tr>
<td title='TestCases.Model_Hero_Image_API_TC.car_basison_ranking()'><b>car_basison_ranking</b><br>Test class: TestCases.Model_Hero_Image_API_TC<br>Parameters: {expectedMessage=, expectedErrorCode=200.0, endpoint=qa/api/subscribe/vehicle/model/1?resolution=hdpi}</td>
<td><div><pre>java.lang.IllegalArgumentException: wrong number of arguments
... Removed 25 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace878725938", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace878725938'><pre>java.lang.IllegalArgumentException: wrong number of arguments
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>0</td>
<td>TestCases.Model_Hero_Image_API_TC@1477089c</td></tr>
<tr>
<td title='TestCases.Customer_Update.customer_Update()'><b>customer_Update</b><br>Test class: TestCases.Customer_Update<br>Parameters: {referralCode=, expectedErrorCode=201, referredBy=, occupationTypeCode=SELF, middleName=Bhardwaj, email=, expectedMessage=null, gender=F, dob=1983-10-26, firstName=Kamini, lastName=, endpoint=qa/customer/api/v1/user/update}, org.testng.TestRunner@7dee835</td>
<td><div><pre>java.lang.AssertionError: expected [Kamini] but found [null]
	at com.MSIL.TestUtils.Verify_Response.verify_Response_data(Verify_Response.java:52)
	at com.MSIL.API_LISTS.Cust_Regsiration_API.customer_update_API(Cust_Regsiration_API.java:59)
	at TestCases.Customer_Update.customer_Update(Customer_Update.java:21)
... Removed 31 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1893180571", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1893180571'><pre>java.lang.AssertionError: expected [Kamini] but found [null]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:129)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:189)
	at org.testng.Assert.assertEquals(Assert.java:199)
	at com.MSIL.TestUtils.Verify_Response.verify_Response_data(Verify_Response.java:52)
	at com.MSIL.API_LISTS.Cust_Regsiration_API.customer_update_API(Cust_Regsiration_API.java:59)
	at TestCases.Customer_Update.customer_Update(Customer_Update.java:21)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>0</td>
<td>TestCases.Customer_Update@7b84fcf8</td></tr>
<tr>
<td title='TestCases.Customer_Update.customer_Update()'><b>customer_Update</b><br>Test class: TestCases.Customer_Update<br>Parameters: {referralCode=, expectedErrorCode=201, referredBy=, occupationTypeCode=SELF, middleName=Bhardwaj, email=, expectedMessage=null, gender=F, dob=1983-10-26, firstName=Kamini, lastName=, endpoint=qa/customer/api/v1/user/update}</td>
<td><div><pre>java.lang.IllegalArgumentException: wrong number of arguments
... Removed 25 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1129000460", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1129000460'><pre>java.lang.IllegalArgumentException: wrong number of arguments
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>0</td>
<td>TestCases.Customer_Update@7b84fcf8</td></tr>
<tr>
<td title='TestCases.Fectch_Nearest_Dealer.fetch_nearest_dealer()'><b>fetch_nearest_dealer</b><br>Test class: TestCases.Fectch_Nearest_Dealer<br>Parameters: {expectedMessage=, expectedErrorCode=200.0, pinCode=324324, longitude=72.454545, citycode=1, endpoint=qa/api/partner/nearest-dealers, distance=50000, latitude=19.11454}, org.testng.TestRunner@7dee835</td>
<td><div><pre>java.lang.NullPointerException
	at com.MSIL.Response.Verify_Response_Nearest_Dealer.verify_Response(Verify_Response_Nearest_Dealer.java:23)
	at com.MSIL.API_LISTS.Fetch_Nearest_Dealer_API.nearest_dealer_API(Fetch_Nearest_Dealer_API.java:36)
	at TestCases.Fectch_Nearest_Dealer.fetch_nearest_dealer(Fectch_Nearest_Dealer.java:21)
... Removed 25 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace224447405", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace224447405'><pre>java.lang.NullPointerException
	at com.MSIL.Response.Verify_Response_Nearest_Dealer.verify_Response(Verify_Response_Nearest_Dealer.java:23)
	at com.MSIL.API_LISTS.Fetch_Nearest_Dealer_API.nearest_dealer_API(Fetch_Nearest_Dealer_API.java:36)
	at TestCases.Fectch_Nearest_Dealer.fetch_nearest_dealer(Fectch_Nearest_Dealer.java:21)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>0</td>
<td>TestCases.Fectch_Nearest_Dealer@44b3606b</td></tr>
<tr>
<td title='TestCases.Fectch_Nearest_Dealer.fetch_nearest_dealer()'><b>fetch_nearest_dealer</b><br>Test class: TestCases.Fectch_Nearest_Dealer<br>Parameters: {expectedMessage=, expectedErrorCode=200.0, pinCode=324324, longitude=72.454545, citycode=1, endpoint=qa/api/partner/nearest-dealers, distance=50000, latitude=19.11454}</td>
<td><div><pre>java.lang.IllegalArgumentException: wrong number of arguments
... Removed 25 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1963663371", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1963663371'><pre>java.lang.IllegalArgumentException: wrong number of arguments
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>0</td>
<td>TestCases.Fectch_Nearest_Dealer@44b3606b</td></tr>
<tr>
<td title='TestCases.Get_Customer_Detail_BY_UUID.get_customer_detail_uuid()'><b>get_customer_detail_uuid</b><br>Test class: TestCases.Get_Customer_Detail_BY_UUID<br>Parameters: {expectedMessage=, expectedErrorCode=200, endpoint=qa/customer/api/v1/user/details/:uuid=d9b97efd-69b7-4e6b-8d10-f4b96a81d9ed}, org.testng.TestRunner@7dee835</td>
<td><div><pre>java.lang.AssertionError: expected [200] but found [403]
	at com.MSIL.API_LISTS.Cust_Regsiration_API.get_customer_details_uuid_API(Cust_Regsiration_API.java:108)
	at TestCases.Get_Customer_Detail_BY_UUID.get_customer_detail_uuid(Get_Customer_Detail_BY_UUID.java:21)
... Removed 31 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace432039778", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace432039778'><pre>java.lang.AssertionError: expected [200] but found [403]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:388)
	at org.testng.Assert.assertEquals(Assert.java:398)
	at com.MSIL.API_LISTS.Cust_Regsiration_API.get_customer_details_uuid_API(Cust_Regsiration_API.java:108)
	at TestCases.Get_Customer_Detail_BY_UUID.get_customer_detail_uuid(Get_Customer_Detail_BY_UUID.java:21)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>0</td>
<td>TestCases.Get_Customer_Detail_BY_UUID@56f0cc85</td></tr>
<tr>
<td title='TestCases.Get_Customer_Detail_BY_UUID.get_customer_detail_uuid()'><b>get_customer_detail_uuid</b><br>Test class: TestCases.Get_Customer_Detail_BY_UUID<br>Parameters: {expectedMessage=, expectedErrorCode=200, endpoint=qa/customer/api/v1/user/details/:uuid=d9b97efd-69b7-4e6b-8d10-f4b96a81d9ed}</td>
<td><div><pre>java.lang.IllegalArgumentException: wrong number of arguments
... Removed 25 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace611621587", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace611621587'><pre>java.lang.IllegalArgumentException: wrong number of arguments
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>0</td>
<td>TestCases.Get_Customer_Detail_BY_UUID@56f0cc85</td></tr>
<tr>
<td title='TestCases.Customer_Registartion.invalid_customer_registration()'><b>invalid_customer_registration</b><br>Test class: TestCases.Customer_Registartion<br>Parameters: {mobile=1, businessType=SUBSCRIBE, referralCode=, expectedErrorCode=400, referredBy=, occupationTypeCode=SELF, middleName=Kumar, email=gurender.kush@nagarro.com, expectedMessage=JSON parse error: Cannot deserialize value of type `java.time.LocalDate` from String \"26-10-1983\": Failed to deserialize java.time.LocalDate: (java.time.format.DateTimeParseException) Text '26-10-1983' could not be parsed at index 5; nested exception is com.fasterxml.jackson.databind.exc.InvalidFormatException: Cannot deserialize value of type `java.time.LocalDate` from String \"26-10-1983\": Failed to deserialize java.time.LocalDate: (java.time.format.DateTimeParseException) Text '26-10-1983' could not be parsed at index 5\n at [Source: (PushbackInputStream); line: 8, column: 12] (through reference chain: com.marutisuzuki.customer.dto.request.CustomerRegisterReqDto[\"dob\"]), gender=, dob=26-10-1983, firstName=Gurender, lastName=Kush, endpoint=qa/customer/api/v1/register}, org.testng.TestRunner@7dee835</td>
<td><div><pre>java.lang.AssertionError: expected [JSON parse error: Cannot deserialize value of type `java.time.LocalDate` from String &quot;26-10-1983&quot;: Failed to deserialize java.time.LocalDate: (java.time.format.DateTimeParseException) Text &apos;26-10-1983&apos; could not be parsed at index 5; nested exception is com.fasterxml.jackson.databind.exc.InvalidFormatException: Cannot deserialize value of type `java.time.LocalDate` from String &quot;26-10-1983&quot;: Failed to deserialize java.time.LocalDate: (java.time.format.DateTimeParseException) Text &apos;26-10-1983&apos; could not be parsed at index 5n at [Source: (PushbackInputStream); line: 8, column: 12] (through reference chain: com.marutisuzuki.customer.dto.request.CustomerRegisterReqDto[&quot;dob&quot;])] but found [JSON parse error: Cannot deserialize value of type `java.time.LocalDate` from String &quot;26-10-1983&quot;: Failed to deserialize java.time.LocalDate: (java.time.format.DateTimeParseException) Text &apos;26-10-1983&apos; could not be parsed at index 0; nested exception is com.fasterxml.jackson.databind.exc.InvalidFormatException: Cannot deserialize value of type `java.time.LocalDate` from String &quot;26-10-1983&quot;: Failed to deserialize java.time.LocalDate: (java.time.format.DateTimeParseException) Text &apos;26-10-1983&apos; could not be parsed at index 0
 at [Source: (PushbackInputStream); line: 1, column: 128] (through reference chain: com.marutisuzuki.customer.dto.request.CustomerRegisterReqDto[&quot;dob&quot;])]
	at com.MSIL.TestUtils.Verify_Response.verify_Response_1(Verify_Response.java:38)
	at com.MSIL.API_LISTS.Cust_Regsiration_API.customer_registration_API(Cust_Regsiration_API.java:40)
	at TestCases.Customer_Registartion.invalid_customer_registration(Customer_Registartion.java:33)
... Removed 31 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace271588719", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace271588719'><pre>java.lang.AssertionError: expected [JSON parse error: Cannot deserialize value of type `java.time.LocalDate` from String &quot;26-10-1983&quot;: Failed to deserialize java.time.LocalDate: (java.time.format.DateTimeParseException) Text &apos;26-10-1983&apos; could not be parsed at index 5; nested exception is com.fasterxml.jackson.databind.exc.InvalidFormatException: Cannot deserialize value of type `java.time.LocalDate` from String &quot;26-10-1983&quot;: Failed to deserialize java.time.LocalDate: (java.time.format.DateTimeParseException) Text &apos;26-10-1983&apos; could not be parsed at index 5n at [Source: (PushbackInputStream); line: 8, column: 12] (through reference chain: com.marutisuzuki.customer.dto.request.CustomerRegisterReqDto[&quot;dob&quot;])] but found [JSON parse error: Cannot deserialize value of type `java.time.LocalDate` from String &quot;26-10-1983&quot;: Failed to deserialize java.time.LocalDate: (java.time.format.DateTimeParseException) Text &apos;26-10-1983&apos; could not be parsed at index 0; nested exception is com.fasterxml.jackson.databind.exc.InvalidFormatException: Cannot deserialize value of type `java.time.LocalDate` from String &quot;26-10-1983&quot;: Failed to deserialize java.time.LocalDate: (java.time.format.DateTimeParseException) Text &apos;26-10-1983&apos; could not be parsed at index 0
 at [Source: (PushbackInputStream); line: 1, column: 128] (through reference chain: com.marutisuzuki.customer.dto.request.CustomerRegisterReqDto[&quot;dob&quot;])]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:189)
	at org.testng.Assert.assertEquals(Assert.java:199)
	at com.MSIL.TestUtils.Verify_Response.verify_Response_1(Verify_Response.java:38)
	at com.MSIL.API_LISTS.Cust_Regsiration_API.customer_registration_API(Cust_Regsiration_API.java:40)
	at TestCases.Customer_Registartion.invalid_customer_registration(Customer_Registartion.java:33)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>1</td>
<td>TestCases.Customer_Registartion@628c4ac0</td></tr>
<tr>
<td title='TestCases.Customer_Registartion.invalid_customer_registration()'><b>invalid_customer_registration</b><br>Test class: TestCases.Customer_Registartion<br>Parameters: {mobile=1, businessType=SUBSCRIBE, referralCode=, expectedErrorCode=400, referredBy=, occupationTypeCode=SELF, middleName=Kumar, email=gurender.kush@nagarro.com, expectedMessage=JSON parse error: Cannot deserialize value of type `java.time.LocalDate` from String \"26-10-1983\": Failed to deserialize java.time.LocalDate: (java.time.format.DateTimeParseException) Text '26-10-1983' could not be parsed at index 5; nested exception is com.fasterxml.jackson.databind.exc.InvalidFormatException: Cannot deserialize value of type `java.time.LocalDate` from String \"26-10-1983\": Failed to deserialize java.time.LocalDate: (java.time.format.DateTimeParseException) Text '26-10-1983' could not be parsed at index 5\n at [Source: (PushbackInputStream); line: 8, column: 12] (through reference chain: com.marutisuzuki.customer.dto.request.CustomerRegisterReqDto[\"dob\"]), gender=, dob=26-10-1983, firstName=Gurender, lastName=Kush, endpoint=qa/customer/api/v1/register}</td>
<td><div><pre>java.lang.IllegalArgumentException: wrong number of arguments
... Removed 25 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1004866348", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1004866348'><pre>java.lang.IllegalArgumentException: wrong number of arguments
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>0</td>
<td>TestCases.Customer_Registartion@628c4ac0</td></tr>
<tr>
<td title='TestCases.Customer_Update.invalid_customer_update()'><b>invalid_customer_update</b><br>Test class: TestCases.Customer_Update<br>Parameters: {referralCode=, expectedErrorCode=400, referredBy=, occupationTypeCode=SELF, middleName=Bhardwaj, email=, expectedMessage=First Name is mandatory, gender=F, dob=1983-10-26, firstName=, lastName=, endpoint=qa/customer/api/v1/user/update}, org.testng.TestRunner@7dee835</td>
<td><div><pre>java.lang.NullPointerException
	at com.MSIL.TestUtils.Verify_Response.verify_Response_data_1(Verify_Response.java:65)
	at com.MSIL.API_LISTS.Cust_Regsiration_API.customer_update_API(Cust_Regsiration_API.java:62)
	at TestCases.Customer_Update.invalid_customer_update(Customer_Update.java:32)
... Removed 25 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace1383367438", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace1383367438'><pre>java.lang.NullPointerException
	at com.MSIL.TestUtils.Verify_Response.verify_Response_data_1(Verify_Response.java:65)
	at com.MSIL.API_LISTS.Cust_Regsiration_API.customer_update_API(Cust_Regsiration_API.java:62)
	at TestCases.Customer_Update.invalid_customer_update(Customer_Update.java:32)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>0</td>
<td>TestCases.Customer_Update@7b84fcf8</td></tr>
<tr>
<td title='TestCases.Customer_Update.invalid_customer_update()'><b>invalid_customer_update</b><br>Test class: TestCases.Customer_Update<br>Parameters: {referralCode=, expectedErrorCode=400, referredBy=, occupationTypeCode=SELF, middleName=Bhardwaj, email=, expectedMessage=First Name is mandatory, gender=F, dob=1983-10-26, firstName=, lastName=, endpoint=qa/customer/api/v1/user/update}</td>
<td><div><pre>java.lang.IllegalArgumentException: wrong number of arguments
... Removed 25 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace661300031", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace661300031'><pre>java.lang.IllegalArgumentException: wrong number of arguments
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>0</td>
<td>TestCases.Customer_Update@7b84fcf8</td></tr>
<tr>
<td title='TestCases.Leasing_Partner_Ranking.leasing_ranking()'><b>leasing_ranking</b><br>Test class: TestCases.Leasing_Partner_Ranking<br>Parameters: {expectedMessage=, expectedErrorCode=200.0, endpoint=qa/api/partner/leasing-company-rating}, org.testng.TestRunner@7dee835</td>
<td><div><pre>java.lang.NullPointerException
	at com.MSIL.Response.Verify_Response_Leasing_Ranking.verify_Response(Verify_Response_Leasing_Ranking.java:24)
	at com.MSIL.API_LISTS.Leasing_Partner_Ranking_API.leasing_partner_ranking(Leasing_Partner_Ranking_API.java:33)
	at TestCases.Leasing_Partner_Ranking.leasing_ranking(Leasing_Partner_Ranking.java:22)
... Removed 25 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace506758159", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace506758159'><pre>java.lang.NullPointerException
	at com.MSIL.Response.Verify_Response_Leasing_Ranking.verify_Response(Verify_Response_Leasing_Ranking.java:24)
	at com.MSIL.API_LISTS.Leasing_Partner_Ranking_API.leasing_partner_ranking(Leasing_Partner_Ranking_API.java:33)
	at TestCases.Leasing_Partner_Ranking.leasing_ranking(Leasing_Partner_Ranking.java:22)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>1</td>
<td>TestCases.Leasing_Partner_Ranking@2cc44ad</td></tr>
<tr>
<td title='TestCases.Leasing_Partner_Ranking.leasing_ranking()'><b>leasing_ranking</b><br>Test class: TestCases.Leasing_Partner_Ranking<br>Parameters: {expectedMessage=, expectedErrorCode=200.0, endpoint=qa/api/partner/leasing-company-rating}</td>
<td><div><pre>java.lang.IllegalArgumentException: wrong number of arguments
... Removed 25 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace731626152", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace731626152'><pre>java.lang.IllegalArgumentException: wrong number of arguments
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>0</td>
<td>TestCases.Leasing_Partner_Ranking@2cc44ad</td></tr>
<tr>
<td title='TestCases.Fectch_Nearest_Dealer.nearest_dealer_not_found()'><b>nearest_dealer_not_found</b><br>Test class: TestCases.Fectch_Nearest_Dealer<br>Parameters: {expectedMessage=Dealer not found., expectedErrorCode=400, pinCode=324324, longitude=72.454545, citycode=1, endpoint=qa/api/partner/nearest-dealers, distance=50, latitude=19.11454}, org.testng.TestRunner@7dee835</td>
<td><div><pre>java.lang.NullPointerException
	at com.MSIL.Response.Verify_Response_Nearest_Dealer.verify_Response_dealernotfound(Verify_Response_Nearest_Dealer.java:47)
	at com.MSIL.API_LISTS.Fetch_Nearest_Dealer_API.nearest_dealer_API(Fetch_Nearest_Dealer_API.java:39)
	at TestCases.Fectch_Nearest_Dealer.nearest_dealer_not_found(Fectch_Nearest_Dealer.java:31)
... Removed 25 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace998575031", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace998575031'><pre>java.lang.NullPointerException
	at com.MSIL.Response.Verify_Response_Nearest_Dealer.verify_Response_dealernotfound(Verify_Response_Nearest_Dealer.java:47)
	at com.MSIL.API_LISTS.Fetch_Nearest_Dealer_API.nearest_dealer_API(Fetch_Nearest_Dealer_API.java:39)
	at TestCases.Fectch_Nearest_Dealer.nearest_dealer_not_found(Fectch_Nearest_Dealer.java:31)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>0</td>
<td>TestCases.Fectch_Nearest_Dealer@44b3606b</td></tr>
<tr>
<td title='TestCases.Fectch_Nearest_Dealer.nearest_dealer_not_found()'><b>nearest_dealer_not_found</b><br>Test class: TestCases.Fectch_Nearest_Dealer<br>Parameters: {expectedMessage=Dealer not found., expectedErrorCode=400, pinCode=324324, longitude=72.454545, citycode=1, endpoint=qa/api/partner/nearest-dealers, distance=50, latitude=19.11454}</td>
<td><div><pre>java.lang.IllegalArgumentException: wrong number of arguments
... Removed 25 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace70104063", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace70104063'><pre>java.lang.IllegalArgumentException: wrong number of arguments
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
</pre></div></td>
<td>0</td>
<td>TestCases.Fectch_Nearest_Dealer@44b3606b</td></tr>
</table><p>
</body>
</html>